packages/augur-core/source/contracts/MKRShutdownHandler.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity 0.5.10;
+
+import 'ROOT/ICash.sol';
+import 'ROOT/external/IDaiVat.sol';
+import 'ROOT/libraries/math/SafeMathUint256.sol';
+
+
+contract MKRShutdownHandler {
```

Comment:
Consider renaming this to something like `CashSender`.  Looking over the way this is used, it appears to basically be a base class for "things that need to be able to send cash".  The current naming is confusing because when I say to myself, "DisputeWindow is a MKRShutdownHandler" that doesn't really feel right, but "DisputeWindow is a CashSender" does.